modules/op-resolver-tasks-step-actions-about.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: CONCEPT
+[id="resolver-tasks-step-actions-about_{context}"]
+
+= About non-versioned and versioned tasks and step actions
+
+The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates `buildah-1-16-0` versioned task, `buildah` non-versioned task, `git-clone-1-16-0` versioned `StepAction` definition, and `git-clone` non-versioned `StepAction` definition.
+
+Both, non-versioned and versioned tasks and step actions have the same metadata, behavior, and specifications, including `params`, `workspaces`, and `steps`. However, they behave differently when you disable them or upgrade the Operator.
+
+Before adopting non-versioned or versioned tasks and step actions as a standard in production environments, cluster administrators might consider their advantages and disadvantages.
+
+.Advantages and disadvantages of non-versioned and versioned tasks and step actions
+[options="header"]
+|===
+
+|  | Advantages | Disadvantages
+
+s| Non-versioned tasks and step actions
+a| 
+* If you prefer deploying pipelines with the latest updates and bug fixes, use non-versioned tasks and step actions. 
+* Upgrading the Operator upgrades the non-versioned tasks and step actions, which consumes fewer resources than multiple versioned tasks and step actions.  
+a| If you deploy pipelines that use non-versioned tasks and step actions, they might break after an Operator upgrade if the automatically upgraded tasks and step actions are not backward-compatible. 
```

Comment:
Maybe just to keep things uniform-looking within the etable, use a bullet here too even tho it's just the 1 item?
```suggestion
a|
* If you deploy pipelines that use non-versioned tasks and step actions, they might break after an Operator upgrade if the automatically upgraded tasks and step actions are not backward-compatible. 
```